ui/app/appointments/views/manage/list/listView.html

```diff
@@ -35,13 +35,16 @@
                         </li>
                     </ul>
                 </td>
+                <td>{{::appointment.additionalInfo["Emergency contact name"]}}</td>
```

Comment:
@rubailly The column titles don't align with the column values, please fix:

![appointmentListColumns](https://user-images.githubusercontent.com/1198827/60424223-26fbc000-9bf0-11e9-902a-459ef23cc0a3.png)

Reply:
You might have missed the CSS when compiling the code. [This commit ](https://github.com/jembi/cameroon-openmrs-module-bahmniapps/pull/99/commits/c3de0fcb33312736be69fb40e444bd939e36572b) is actually the one adjusting the column titles with their corresponding values.
![Screenshot from 2019-07-01 11-35-07](https://user-images.githubusercontent.com/840071/60426518-19950480-9bf5-11e9-8a46-6ca7fb73b6a6.png)